### ISSUE TITLE
New Syntax?

### ISSUE THREAD

**Issue by Quantumplation** [nikita-volkov/record#23]
Did you change the syntax in the latest release?  I can't find any examples of it anywhere.  A Readme with (up to date) examples would be super helpful!

**Comment by nikita-volkov** [comment]
Yes. The library now only provides the types, while the syntax is handled by the ["record-preprocessor"](http://hackage.haskell.org/package/record-preprocessor). Check it out for the basic yet exhaustive demo.

The detailed documentation and tutorial are coming in a week or two, when I fix the error-reporting issues in the preprocessor and make an official release.

**Comment by Quantumplation** [comment]
Sure! I ended up figuring it out after a few hours of diddling last night. I had seen the demo, but I wasn't familiar enough with Haskell to know how to configure cabal to use a preprocessor.  I eventually got it though.  Absolutely love this library! Saves an insane amount of headaches, thank you!

**Comment by rabipelais** [comment]
Can ghci (or cabal-repl) be configured to use the preprocessor? To play around with the records in ghci.

**Comment by nikita-volkov** [comment]
@rabipelais Not yet. But maybe soon.

**Comment by dimsuz** [comment]
@nikita-volkov that blog post you wrote about is still coming or I missed it somehow? (checked your blog, didn't find recent posts related to `record`)

EDIT: Oh, I mean "the detailed documentation and tutorial are coming...", not the blog post :)

**Comment by nikita-volkov** [comment]
@dimsuz Still in the making. Been too busy lately.

**Comment by dimsuz** [comment]
Looking forward to it :)

пн, 21 сент. 2015, 16:04, Nikita Volkov notifications@github.com:
